specification/cosmos-db/resource-manager/readme.md

```diff
@@ -39,6 +39,7 @@ input-file:
 - Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json
```

Comment:
should the tag be updated to use this package instead of 2020-04?

Reply:
Let us keep the stable apiVersions as our defaults everywhere, the features in the preview apiVersions(PITR, RBAC, sqlx) are all in private preview right now.